bin/comBGC.py

```diff
@@ -0,0 +1,521 @@
+#!/usr/bin/env python3
+
+from Bio import SeqIO
+import pandas as pd
+import argparse
+import os
+import re
+
+welcome = """\
+                ........................
+                    * comBGC v.0.5 *
+                ........................
+    This tool aggregates the results of BGC prediction tools:
+                antiSMASH, deepBGC, and GECCO
+           For detailed usage documentation please
+             refer to https://nf-co.re/funcscan
+    ........................................................."""
+
+# Initialize parser
+parser = argparse.ArgumentParser(
+    prog="comBGC", formatter_class=argparse.RawDescriptionHelpFormatter, description=(welcome), add_help=True
+)
+
+# Input options
+parser.add_argument(
+    "-i",
+    "--input",
+    metavar="PATH(s)",
+    dest="input",
+    nargs="*",
+    help="path(s) to the required output file(s) of antiSMASH, DeepBGC and/or GECCO",
+)
+parser.add_argument(
+    "-o",
+    "--outdir",
+    metavar="PATH",
+    dest="outdir",
+    nargs="?",
+    help="directory for comBGC output. Default: current directory",
+    type=str,
+    default=".",
+)
+parser.add_argument("-vv", "--verbose", help="increase output verbosity", action="store_true")
+parser.add_argument("-v", "--version", help="print version number and exit", action="store_true")
+
+# Get command line arguments
+args = parser.parse_args()
+
+# Assign input arguments to variables
+input = args.input
+outdir = args.outdir
+verbose = args.verbose
+version = args.version
+
+if version:
+    exit("comBGC 0.5")
+
+input_antismash = []
+input_deepbgc = []
+input_gecco = []
+
+# Assign input files to respective tools
+for path in input:
+    if path.endswith(".gbk"):
+        if re.search(".*_cluster_.+\.gbk", path):
+            input_gecco.append(path)
+        else:
+            input_antismash.append(path)
+    elif path.endswith("bgc.tsv"):
+        input_deepbgc = path
+    elif path.endswith("clusters.tsv"):
+        input_gecco.append(path)
+    elif path.endswith("knownclusterblast/"):
+        input_antismash.append(path)
+
+# Make sure that at least one input argument is given
+if not (input_antismash or input_gecco or input_deepbgc):
+    exit("Please specify at least one input directory (--antismash, --deepbgc, --gecco) or see --help")
+if not outdir:
+    exit("Please specify an output directory (--outdir) or see --help")
+
```

Comment:
Can be deleted now?

Reply:
I'd keep the first if statement (in case something goes wrong with the inputs, the exit message might be helpful). Output message deleted since it can never be evoked